src/metabase/models/data_permissions.clj

```diff
@@ -608,8 +611,7 @@
   "Sets a single permission to a specified value for a given group and database. If a permission value already exists
   for the specified group and object, it will be updated to the new value.
 
-  Block permissions (i.e. :perms/view-data :blocked) can only be set at the database-level, despite :perms/view-data
-  being a table-level permission."
+  Block permissions (i.e. :perms/view-data :blocked) can be set at the table or database-level."
```

Comment:
I'd support just removing this line — previously it was calling out a special case, which is no longer special

Reply:
Done